### PR TITLE
Session progress cards

### PR DESCRIPTION
**Pull request by stefan-philip**
Currently does not include the images for each card

### CODE REVIEW

File: frontend/src/pages/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -154,6 +159,66 @@ const SessionContainer = styled.div`
...
+    if (isComplete) {
+      return `
+    background: ${SKY} !important;
+  `;
+    } else {
+      return `
+    background: ${INCOMPLETE_GRAY} !important; 
+  `;
+    }
+  }};
```
Comment (jrosen081) [inline frontend/src/pages/Dashboard/Dashboard.tsx]: Can we convert this to be a bit smaller, like the suggestion? (can't promise if the syntax is correct or not, but that vibe)
```suggestion
  background: $({isComplete}: SessionCompletionProp) => isComplete ? SKY : INCOMPLETE_GRAY} !important
```

File: frontend/src/pages/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -485,10 +571,74 @@ const Dashboard: FunctionComponent<DashboardParams> = ({
             />
 
             <TitleText>list of sessions</TitleText>
+
+            <SessionCardContainer>
+              {/* Will probably be cleaner once we do a .map to create
+              a SessionCard // It will be easier to do that once we
+              have an array of image links // For each session card
+              */}
```
Comment (jrosen081) [inline frontend/src/pages/Dashboard/Dashboard.tsx]: Are the images not ready or something?

**Review by jrosen081 — APPROVED** [review]
Looks awesome. One small comment and a question about images and then :shipit:

File: frontend/src/pages/Dashboard/Dashboard.tsx (hunk 1 above)
Comment (stefan-philip) [reply frontend/src/pages/Dashboard/Dashboard.tsx]: Yes, thanks!

**Review by jrosen081 — APPROVED** [review]
Looks amazing!

**Pull request merged**